### PR TITLE
Alex tutorials - Getting Started

### PR DESCRIPTION
**Pull request by alexandercyu**
Getting Started section
- gives short general overview of key sections of the Marble Climate site
- gives short description of how to find nodes and log in
- give short description of creating an account, changing password, account management

### CODE REVIEW

**Review by mishaschwartz — CHANGES_REQUESTED** [review]
This page should be the place that users can go to get started, not a mix of account management, jupyterlab, etc.

Please reconfigure this:

- 1 page that describes how the marble platform is configured and what it means to access a node on the network. Include how to look up information about each node on the marbleclimate.com site.
  - you can mention things like the IDE (jupyterlab) and the Datacatalog (STAC) but leave any description of how to use those pages to a different tutorial to be written later 
- 1 page that describes how to get an account (be generic with specific examples if needed)
- 1 page that describes how to manage account information (ie. how to use magpie as a regular user)

Please let me know if you have any questions about this and we can discuss further

File: tutorials/_toc.yml
Comment (alexandercyu) [reply tutorials/_toc.yml:8]: Changed link to getting-started.md in tutorials/intro.md

**Comment by dchandan** [comment]
I'll get to it soon, I'm sorry for the delay :(

File: tutorials/marble/getting-started.md
Comment (alexandercyu) [reply tutorials/marble/getting-started.md]: Added a short paragraph describing nodes

File: tutorials/marble/getting-started.md
Comment (alexandercyu) [reply tutorials/marble/getting-started.md]: Split off into it's own file

File: tutorials/marble/getting-started.md
Comment (alexandercyu) [reply tutorials/marble/getting-started.md:3]: Removed

File: tutorials/marble/getting-started.md
Comment (alexandercyu) [reply tutorials/marble/getting-started.md]: Moved to its own file

File: tutorials/marble/getting-started.md
Comment (alexandercyu) [reply tutorials/marble/getting-started.md]: It was supposed to just show the account details and the management section would have shown how to change them.  But I guess it is too similar.

Removed section.

File: tutorials/marble/images/getting-started/left-menu-commands-icon.png
Comment (alexandercyu) [reply tutorials/marble/images/getting-started/left-menu-commands-icon.png:1]: Removed

File: tutorials/marble/getting-started.md
Comment (alexandercyu) [reply tutorials/marble/getting-started.md]: Removed from here and put in Jupyter documentation

File: tutorials/marble/getting-started.md
Comment (alexandercyu) [reply tutorials/marble/getting-started.md:60]: Removed Tutorials section

File: tutorials/marble/getting-started.md
Comment (alexandercyu) [reply tutorials/marble/getting-started.md]: Changed

**Comment by alexandercyu** [comment]
> I'll get to it soon, I'm sorry for the delay :(

I think the latest pushes are ready for you to take a look at.

**Review by mickyals — COMMENTED** [review]
All good from my experience with the platform, just slight a reorganization of the paragraph highlighted

File: tutorials/marble/getting-started.md
Comment (alexandercyu) [reply tutorials/marble/getting-started.md:17]: Switched the sections around.

File: tutorials/marble/getting-started.md
Comment (mishaschwartz) [reply tutorials/marble/getting-started.md]: You've remove this but replaced it with a different manually written table of contents above it. 

Please read through the tutorial in the link above and change this so that jupyterbook creates a table of contents for this page for you.

If you are having trouble with the tutorial, let me know and we can chat about it

File: tutorials/marble/account-management.md
Comment (mishaschwartz) [inline tutorials/marble/account-management.md:6]: The `<a>` tags are unnecessary if you let jupyterbook create the table of contents for you

Comment (mishaschwartz) [inline tutorials/marble/account-management.md:6]: Also all pages titles should be at "heading level 1", this means that each page has to be preceded by a single `#`

The documentation for this is here: https://jupyterbook.org/en/stable/structure/sections-headers.html?highlight=page%20title#how-headers-and-sections-map-onto-to-book-structure

File: tutorials/marble/getting-started.md
Comment (mishaschwartz) [inline tutorials/marble/getting-started.md:22]: Please don't write this sort of thing in the tutorials, Every time we add a new node this information will need to be updated. Also the PCIC node isn't even in the network yet so this number is currently inaccurate.

Please remove this whole paragraph. If you need an introductory paragraph please talk about nodes and the network in the abstract without giving specific numbers or descriptions that are still very fluid.

Please change this to something like:

File: tutorials/marble/getting-started.md
Comment (mishaschwartz) [inline tutorials/marble/getting-started.md:13]: It will not be available to the public. It might be if the node admin wants to set it up that way but that is not a requirement or the default configuration.

File: tutorials/marble/getting-started.md
Comment (mishaschwartz) [inline tutorials/marble/getting-started.md:16]: This isn't true either...

File: tutorials/marble/getting-started.md
Comment (mishaschwartz) [inline tutorials/marble/getting-started.md:1]: I'd like to rewrite most of the content in this page please. There's a lot of misleading statements here that advertise things about the network that are not the case.

File: tutorials/marble/images/getting-started/terminal-session-ui.png
Comment (mishaschwartz) [inline tutorials/marble/images/getting-started/terminal-session-ui.png:1]: unused

**Review by mishaschwartz — CHANGES_REQUESTED** [review]
I'm going to make a PR with suggested updates to the content here.

File: tutorials/marble/account-management.md
Comment (alexandercyu) [reply tutorials/marble/account-management.md:6]: I realized that.  I was going to do it in a separate PR once the content stuff was taken care of.

File: tutorials/marble/images/getting-started/terminal-session-ui.png
Comment (alexandercyu) [reply tutorials/marble/images/getting-started/terminal-session-ui.png:1]: Deleted

File: tutorials/marble/account-management.md
Comment (alexandercyu) [reply tutorials/marble/account-management.md:6]: Changed heading to level 1

File: tutorials/marble/getting-started.md
Comment (alexandercyu) [reply tutorials/marble/getting-started.md]: Removed

File: tutorials/marble/getting-started.md
Comment (alexandercyu) [reply tutorials/marble/getting-started.md]: Removed

File: tutorials/marble/getting-started.md
Comment (alexandercyu) [reply tutorials/marble/getting-started.md]: Removed

File: tutorials/marble/images/getting-started/terminal-session-ui.png
Comment (alexandercyu) [reply tutorials/marble/images/getting-started/terminal-session-ui.png:1]: Removed

**Comment by mishaschwartz** [comment]
@alexandercyu see suggested edits here: #20

I've made you a reviewer

**Review by mishaschwartz — APPROVED** [review]
Approved pending feedback from others

**Comment by mishaschwartz** [comment]
@alexandercyu please do not merge this until we get additional feedback from the other reviewers

File: tutorials/marble/account-management.md
Comment (dchandan) [inline tutorials/marble/account-management.md:7]: I thought authorization was done by twitcher.

File: tutorials/marble/explore-the-network.md
Comment (dchandan) [inline tutorials/marble/explore-the-network.md:5]: I suggest removing this local contents section from all pages for two reasons: 1. in Jupyter book, local TOC is anyways shown by default on the right hand side, 2. Adding this TOC makes the section titles look like hyperlinks which are empty, and it doesn't look nice to have hyperlinks in huge font size. See the two before and after figures below.

Screenshot 2024-01-02 at 12 06 40 AM
Screenshot 2024-01-02 at 12 07 07 AM

File: tutorials/marble/explore-the-network.md
Comment (dchandan) [inline tutorials/marble/explore-the-network.md:7]: I think this heading should be removed since this is an introductory paragraph. Otherwise we have a H1 heading "Explore the Network" and right under, a H2 heading "Learn about the network" which sounds redundant.

**Comment by dchandan** [comment]
Proposal: We should have a "Getting Support" page or something like that under "The Marble Platform" section. Here, we could summarize facts such as additional information about services and software components are available from the respective Github links provided on the main webpage, and questions can be asked on the "issues" section of the repos. And we could mention that nodes provide a contact email address, but response is not guaranteed. Eventually we can add reference to our discourse page. I imagine the content and structure of this page will evolve continually as we further refine the idea of what support we can, and are able to provide. 

The "Getting Support" page can be started as a separate PR as well.

**Comment by mishaschwartz** [comment]
> The "Getting Support" page can be started as a separate PR as well.

Good idea, please either make an issue for it or add it to #10

**Comment by dchandan** [comment]
One final organizational comment that comes to mind is about the headings we use and how we organize the content. Based on #10, it seems we wanted to have two top level "chapters" (in Jupyter Book lingo) one for "Users" and one for "Admin", and then we'd sub-divide the content based on the structure in that issue. It seems that with the organization in this PR we are not following that approach. Is this intentional?

**Comment by dchandan** [comment]
Something like this mockup:
mockup

**Comment by mishaschwartz** [comment]
> It seems that with the organization in this PR we are not following that approach. Is this intentional?

Not intentional. The plan was to make changes to the overall organization once we had more content (once #16, #17, and #19 were pulled in). But I'm fine if we start organizing better now

**Comment by dchandan** [comment]
OK, I'm reorganizing this in a branch I've branched from this branch.

File: tutorials/marble/explore-the-network.md
Comment (alexandercyu) [reply tutorials/marble/explore-the-network.md]: Removed the local contents from explore-the-network.md, account-management.md, and get-an-account.md.

**Review by dchandan — APPROVED** [review]

**Pull request merged**
